### ISSUE TITLE
alpine package, make ocaml build dependency

### ISSUE THREAD

**Issue by abate** [ocaml/opam#4479]
the opam package on alpine has a runtime dependency on the ocaml package, 
which I believe this should just be a build-dependency instead. Would it possible
to change the dependency type, therefore making the alpine images
using opam much smaller ?

in alpine lingo is a `makedepends` : https://wiki.alpinelinux.org/wiki/Creating_an_Alpine_package#depends_.26_makedepends

ref: https://pkgs.alpinelinux.org/package/v3.12/community/x86_64/opam

**Comment by rjbou** [comment]
ping @avsm

**Comment by abate** [comment]
I think the fix is super easy :
```11,12c11,12
< depends="ocaml curl tar unzip rsync patch bubblewrap bash"
< makedepends="ocaml-compiler-libs"
---
> depends="curl tar unzip rsync patch bubblewrap bash"
> makedepends="ocaml-compiler-libs ocaml"
```
Unless I missed something ...

**Comment by dra27** [comment]
I expect the intention is that it means that `opam init` doesn't then have to build a compiler (other distros do this).

@avsm - what do you think?

**Comment by kit-ty-kate** [comment]
This was fixed in Alpine 3.14 some years ago. In the future, for this kind of issue, please do open a ticket on https://gitlab.alpinelinux.org/alpine/aports/ instead
